src/main/java/com/knighttodo/knighttodo/gateway/privatedb/representation/Block.java

```diff
@@ -33,7 +32,6 @@
         name = "uuid",
         strategy = "org.hibernate.id.UUIDGenerator"
     )
-    @ColumnDefault("random_uuid()")
```

Comment:
Can you explain it please? Is it related to postgres?

Reply:
Yes, postgres does not support it